### PR TITLE
EZEE-3489 - Update a schema for new entity managers

### PR DESCRIPTION
**Pull request by dabrt**
| Question      | Answer |
| ------------- | --- |
| JIRA Ticket   | EZEE-3489 |
| Versions      | 3.2/3.3 |

This change must be released together with 3.2.6 / 3.3.2 tags.

### CODE REVIEW

**Review by webhdx — APPROVED** [review]
It's okay. We just need to add the same steps for people updating to 2.5.18 and 3.3.2.

File: docs/updating/4_update_3.2.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ If you are updating from a version prior to 3.0, you have to implement all the c
     During database update, you have to go through all the changes between your current version and your final version
     **for example, when you update from v2.2 to v2.5, you have to perform all the steps from: <2.3, <2.4 and <2.5**.
     The database will work properly only if you apply all the required changes.
+
+!!! caution "Updating a schema for entity managers"
+
+    If you are updating to v3.2.6, after you complete the procedure, do the steps described in the [Update entity managers](#update-entity-managers) section to update the GraphQL schema that is generated from your repository.
```
Comment (DominikaK) [inline docs/updating/4_update_3.2.md:13]:
```suggestion
    If you are updating to v3.2.6, after you complete the procedure,
    do the steps described in the [Update entity managers](#update-entity-managers) section
    to update the GraphQL schema that is generated from your Repository.
```

File: docs/updating/4_update_3.2.md [hunk 2]
```diff
@@ -30,6 +34,29 @@ If you are updating from a version prior to 3.0, you have to implement all the c
 
     `psql <database_name> < upgrade/db/postgresql/ezplatform-3.1.0-to-3.2.0.sql`
 
+## Update entity managers
+
+Version v3.2.6 introduces new entity managers.
+To ensure that they work in multi-repository setups, you must update the GraphQL schema.
+You do this manually by following this procedure:
+
+1. Update your project to v3.2.6 and check if the container has been built. If not, run the `php bin/console cache:clear` command to generate the container.
```
Comment (DominikaK) [inline docs/updating/4_update_3.2.md:43]: Can we quickly explain how you "check if the container has been built", or can we assume this is clear for the audience?

File: docs/updating/4_update_3.2.md [hunk 3]
```diff
@@ -30,6 +34,29 @@ If you are updating from a version prior to 3.0, you have to implement all the c
...
+
+1. Run the following command to discover the names of the new entity managers. 
+    Take note of the names that you discover:
+
+    `php bin/console debug:container --parameter=doctrine.entity_managers --format=json | grep ibexa_`
+
+1. For every entity manager prefixed with "ibexa_", run the following command:
```
Comment (DominikaK) [inline docs/updating/4_update_3.2.md:50]:
```suggestion
1. For every entity manager prefixed with `ibexa_`, run the following command:
```

File: docs/updating/4_update_3.2.md [hunk 4]
```diff
@@ -30,6 +34,29 @@ If you are updating from a version prior to 3.0, you have to implement all the c
...
+
+    `php bin/console doctrine:schema:update --em=<ENTITY_MANAGER_NAME> --dump-sql`
+  
+1. Review the queries and ensure that there are no harmful changes that could affect your data.
+
+1. For every entity manager prefixed with "ibexa_", run the following command to run queries on the database:
```
Comment (DominikaK) [inline docs/updating/4_update_3.2.md:56]:
```suggestion
1. For every entity manager prefixed with `ibexa_`, run the following command to run queries on the database:
```

**Review by DominikaK — APPROVED** [review]
Side: please change the PR/commit name to a more descriptive one.

File: docs/updating/4_update_3.2.md (hunk 2 above)
Comment (dabrt) [inline docs/updating/4_update_3.2.md:43]:
```suggestion
1. Update your project to v3.2.6 and run the `php bin/console cache:clear` command to generate the [service container](../guide/service_container.md).
```

**Pull request merged**